modules/apps/commerce/commerce-product-content-web/src/main/resources/META-INF/resources/product_detail/render/js/util/index.js

```diff
@@ -55,7 +55,7 @@ export function updateProductFields(data) {
 	].forEach(fillField);
 
 	stockQuantityContainer.innerHTML = cpInstance.stockQuantity
-		? cpInstance.stockQuantity + Liferay.Language.get('in-stock')
+		? cpInstance.stockQuantity + ' ' + Liferay.Language.get('in-stock')
```

Comment:
the test found that this is actually changing the label used in the single sku case. So please change as follow:
```suggestion
		? sub(Liferay.Language.get('x-in-stock'),  cpInstance.stockQuantity)
```

sub function is imported like this: _import sub from 'frontend-js-web';_ in other places